cognite/experimental/data_classes/transformation_jobs.py

```diff
@@ -137,21 +141,40 @@ def wait(self, polling_interval: float = 1):
                 >>> job2.wait()
                 >>> if TransformationJobStatus.FAILED not in [job1.status, job2.status]:
                 >>>     c.transformations.run(id = 3, wait = False)
+
+            wait transformation for 5 minutes and do something if still running:
+
+                >>> from cognite.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>>
+                >>> job = c.transformations.run(id = 1, wait = False)  
+                >>> if job.wait(timeout = 5.0*60).status not in [TransformationJobStatus.FAILED, TransformationJobStatus.COMPLETED]:
+                >>>     # send notification of long running transformation
```

Comment:
```suggestion
                >>>     # send notification of long running transformation
```
is this leftover from some other comment?

Reply:
It was intended to mean like "here goes the code of whatever you want to do if the transformation is still running, for example, sending a notification".... but didn't find a clear way to put it, other than a comment.

Reply:
changed it a bit, maybe it's more instructive now

Reply:
thanks! it makes sense, sorry for missing the point :D